packages/core/src/Stores/client-store.js

```diff
@@ -634,6 +634,9 @@ export default class ClientStore extends BaseStore {
                 // So it will send an authorize with the accepted token, to be handled by socket-general
                 await BinarySocket.authorize(client.token);
             }
+            runInAction(() => {
+                this.is_populating_account_list = false;
+            });
```

Comment:
What's the rationale behind moving this line here?

Reply:
For MF accounts, real accounts is not supported in deriv. So we authorize again with virtual account token during oauth redirect. This is to make sure that correct authorization is done, so that we can wait for this flag before sending messages like `active_symbols`